migrations/20171104220247_add_heilas_table.js

```diff
@@ -0,0 +1,27 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('heilas', function(table) {
+      table.integer('userId').notNullable().unique().primary().index();
+      table.foreign('userId')
+        .references('id')
+        .inTable('users')
+        .onDelete('RESTRICT')
```

Comment:
by using `RESTRICT`, we make it so that a user can't be deleted, if he / she has a heila. Is this the sort of behaviour we want? Is there a reason we can't just `CASCADE` the deletion as well?

Reply:
No. Mistake. Will fix.

Reply:
@cc-akarvinen